### PR TITLE
Nc news task 5

### PR DESCRIPTION
**Pull request by Northerner988**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const db = require("../db/connection");
 const testData = require("../db/data/test-data/index.js");
 const seed = require("../db/seeds/seed.js");
 const allEndPoints = require("../endpoints.json");
+const { toBeSortedBy } = require("jest-sorted");
```
Comment (MKD01) [inline __tests__/app.test.js:7]: This shouldn't need to be required in to use

File: __tests__/app.test.js [hunk 2]
```diff
@@ -79,3 +80,30 @@ describe("GET/articles/:article_id", () => {
...
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(typeof article.article_img_url).toBe("string");
+          expect(typeof article.comment_count).toBe("number");
+          expect(articles.body).toBe(undefined);
+        });
```
Comment (MKD01) [inline __tests__/app.test.js:106]: Alternatively to this approach a couple matchers you can look into are .objectContaining and .toMatchObject. These also allow for later extension of the endpoint as well making then very flexible matchers.

**Review by MKD01 — APPROVED** [review]
Great work so far, im happy for you to merge this in 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (Northerner988) [reply __tests__/app.test.js:7]: when i remove it, toBeSortedBy no longer works

**Pull request merged**